samples/apps/copilot-chat-app/webapi/Controllers/CopilotChatControllers/ChatController.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Reflection;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Microsoft.Graph;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Reliability;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.Skills.MsGraph;
+using Microsoft.SemanticKernel.Skills.MsGraph.Connectors;
+using Microsoft.SemanticKernel.Skills.MsGraph.Connectors.Client;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+using SemanticKernel.Service.Config;
+using SemanticKernel.Service.Model;
+using SemanticKernel.Service.Skills;
+
+namespace SemanticKernel.Service.Controllers;
+
+/// <summary>
+/// Controller responsible for handling chat messages and responses.
```

Comment:
```suggestion
/// Controller responsible for handling chat messages and responses.
/// This controller utilizes the planner to generate well formed step by step plans 
/// and to connect copilot chat to a variety of plugins.
```

Reply:
Planner can be disabled. The skill/function doesn't necessarily use planner.